org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/interpreter/listeners/WollokRemoteContextStateListener.xtend

```diff
@@ -33,7 +35,10 @@ class WollokRemoteContextStateListener implements XInterpreterListener {
 	override started() {}
 
 	override terminated() {
-		this.detectChanges(!forRepl)
+		if (singleTest === null) {
+			singleTest = false
+		}
+		this.detectChanges(!forRepl && singleTest)
```

Comment:
Esto está bien que cambie el atributo? O como es `terminated()` no importa?

Podría ser `this.detectChanges(!forRepl && singleTest ?: false)`??

Reply:
Es un "por las dudas"... :)

Está bueno el cambio, pero tendría que probarlo bien y me da cosita porque el efecto que tiene es muy poderoso (por ejemplo hizo reventar los asserts)...